### DISCUSSION TITLE
lazy Sync not working when starting nvim in tmux

### DISCUSSION THREAD

**Discussion by erlangparasu** [LazyVim/LazyVim#4278] [Q&A]
lazy Sync not working when starting nvim in tmux. git fetch fail.

nvim: v0.10.1
macos: Ventura v13.6.7

**Comment by dpetka2001** [comment]
Must be on your system only. I just updated mine in a tmux environment.

**Reply by erlangparasu** [reply]
i tried also, success on first attempt (after reinstall tmux). but later, i got that error again.
i think it's tmux issue, i just create issue on tmux repo https://github.com/tmux/tmux/issues/4070

**Reply by dpetka2001** [reply]
For your information I noticed that in the issue you created you use tmux 3.4, whereas I'm still on 3.3a. Maybe worth also checking that out and try to see if you don't encounter the issue on 3.3a? Just a suggestion.
